WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/progress/BackupDownloadProgressFragment.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.jetpack.backup.progress
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import javax.inject.Inject
+
+class BackupDownloadProgressFragment : Fragment(R.layout.backup_download_progress_fragment) {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: BackupDownloadProgressViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val nonNullActivity = requireActivity()
+        (nonNullActivity.application as WordPress).component().inject(this)
```

Comment:
Idea (💡 ): These two lines can be extracted into a descriptive one-liner `initDagger` private function:
```
    private fun initDagger() {
        (requireActivity().application as WordPress).component().inject(this)
    }
```
Wdyt?

Reply:
@ParaskP7 - I'm on the fence about this one. (A) because we mention Dagger and yes, although we use it exclusively today, we might not in the future so I don't want to litter the code with references to an external library (B) It seems like overkill to put this one line in its own fun. Another good candidate for the team to discuss in slack.

Reply:
Thanks @zwarm ! 🌟 

1) I agree on the logic that `Dagger` is too specific, I was also thinking about calling that `initDi()` or something like that, but I end up going with the `initDagger()` suggestion because actually this is a code line specific to `Dagger`, not an interface we can swap and use another framework instead. All `DI` frameworks I used so far have their own way of dealing with dependency injection.
2) It seems indeed that it is an overkill, I agree. How I think about it is that having `initDagger()` instead of this more complex inject dependency injection related line, will make it easier for me to read the code without needing to always read it again and again. By reading `initDagger()` I will not put an effort next time in reading the whole line trying to figure out if something is missing.

PS: This is just conversation for conversation sake, and it is very subjective, so please treat it as such. 🙏

Reply:
> I will not put an effort next time in reading the whole line trying to figure out if something is missing.

I can go along with this reasoning - especially if we all agree to move it. See commit efa3a68